packages/plugins/LoadingIndicator/src/store/mutations.ts

```diff
@@ -0,0 +1,29 @@
+import { generateSimpleMutations } from '@repositoryname/vuex-generators'
+import { LoaderStyles } from '@polar/lib-custom-types'
+import { LoadingIndicatorMutations } from '../types'
+import { getInitialState } from './state'
+
+const mutations = {
+  ...generateSimpleMutations(getInitialState()),
+  setLoaderStyle(state, style: LoaderStyles) {
+    if (style) {
+      if (Object.values(LoaderStyles).includes(style)) {
```

Comment:
```suggestion
const styles = [
  'CircleLoader',
  'DefaultLoader',
  'none',
  'RingLoader',
  'RollerLoader',
  'SpinnerLoader',
  'v-progress-linear',
]

const mutations = {
  ...generateSimpleMutations(getInitialState()),
  setLoaderStyle(state, style?: LoaderStyles) {
    if (style) {
      if (styles.includes(style)) {
```
Now I see why you used an enum :D
However, I still believe that using the suggested type is more fitting thus this suggestion here. I don't feel overly pleased with it though, so feel free to implement it differently here.